### DISCUSSION TITLE
cc2531 on Home Assistant with zigbee2mqtt

### DISCUSSION THREAD

**Discussion by yoda98mnt** [Koenkk/zigbee2mqtt#16224] [Devices]
I habe a cc2531 and the FP1 the Software on the cc2531 ist this: https://github.com/Koenkk/Z-Stack-firmware/blob/Z-Stack_Home_1.2_20211115/20211116/coordinator/Z-Stack_Home_1.2/bin/default/CC2531_DEFAULT_20211115.zip
I can see the FP1 at home assistant but I have this message: 2023-01-15 09:08:55: Received message from unsupported device with Zigbee model ‘lumi.motion.ac01’ and manufacturer name ‘aqara’ so I can’t use the FP1. The hole sytem from Home assistant is on the newest Version. The coordinator Channel is 15 and I am using Home Assistent with zigbee2mqtt. Can someone help me to fix the Problem?

**Comment by lux73** [comment]
i can't help sorry - but in my environment it is working with CC2652RB and Firmware 20221226 / z2m 1.29.2 (Channel 25)

generally your cc2531 is approved as weak - please consider to by a new powerful Stick with CC2652x Chipset
